src/react/components/pages/editorPage/editorPage.tsx

```diff
@@ -129,6 +138,36 @@ export default class EditorPage extends React.Component<IEditorPageProps, IEdito
             const project = this.props.recentProjects.find((project) => project.id === projectId);
             await this.props.actions.loadProject(project);
         }
+
+        let modelPath = "";
+        if (this.props.project.activeLearningSettings.modelPathType === ModelPathType.Coco) {
+            if (isElectron()) {
+                const appPath = this.getAppPath();
+
+                if (Env.get() !== "production") {
+                    modelPath = appPath + "/cocoSSDModel";
+                } else {
+                    modelPath = appPath + "/../../cocoSSDModel";
+                }
+            } else {
+                modelPath = "https://jmangiadiag.blob.core.windows.net/vottcontainer";
```

Comment:
I don't have a great answer here, but we have to be careful about hosting things ourselves, since we can't guarantee it's always going to be there. 

A couple thoughts: 
- can/should we embed this in the SPA, version it just like any other asset
- Is there a better 'public' place for this to live?

Reply:
My proposal in the mail was exactly to deploy this in the SPA but we need to talk about if we need to make some change to the pipeline.